### ISSUE TITLE
Native ES6 Module Support

### ISSUE THREAD

**Issue by foundling** [kriszyp/alkali#60]
Hey Kris, are you open to a PR augmenting the way alkali is packaged to include native ES6 module support?

**Comment by kriszyp** [comment]
Yes, that sounds great. How would we go about doing this? Would it mean generating a .mjs file in the build? If that could provides native module support, that would be awesome.

**Comment by foundling** [comment]
My original thought was that it would be really nice to able to use the `type="module` attribute on the script tag to access `alkali` as a module without build tools.  Supporting Node is equally important of course, but more complicated, so I'll do some research to see what the current state of Node.js ES module support is today.

**Comment by kriszyp** [comment]
I wonder if switching to Rollup might offer better support for building ES6 modules as output?
